utils/build/docker/dotnet_http/parametric/ApmTestApi.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <InvariantGlobalization>true</InvariantGlobalization>
+    <UserSecretsId>f56b479c-4bfa-4286-bb36-96a6273695e3</UserSecretsId>
+    <Configuration Condition="'$(Configuration)' == ''">Development</Configuration>
```

Comment:
I think we should remove this line. The Configuration property tends to be `Debug` or `Release` so we don't need to mess with it, unless you had a specific reason in mind